Frends.MicrosoftSQL.BulkInsert/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.1] - 2023-01-26
```

Comment:
Parameters of the task have changed, so you should increase the minor version instead of build version.